jobs/config.json

```diff
@@ -3354,6 +3354,22 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-gce-pod-preemption-beta": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gce-pod-preemption.env",
+      "--gcp-project=openshift-gce-devel",
```

Comment:
is this project any special?

Reply:
Sorry, was testing it with that project. Will remove it.